app/src/main/java/org/oppia/app/player/state/StateAdapter.kt

```diff
@@ -8,13 +8,17 @@ import androidx.databinding.ViewDataBinding
 import androidx.recyclerview.widget.RecyclerView
 import androidx.databinding.library.baseAdapters.BR
 import kotlinx.android.synthetic.main.content_item.view.*
+import kotlinx.android.synthetic.main.selection_interaction_item.view.*
```

Comment:
Nit: please don't wildcard import (ditto for other wildcard imports already here).

Reply:
done.